client/src/components/LogoHeader/LogoHeader.tsx

```diff
@@ -0,0 +1,23 @@
+import useStyles from './useStyles';
+import Box from '@material-ui/core/Box';
+import logo from '../../../src/Images/logo.png';
+import { Link as RouterLink } from 'react-router-dom';
+import { Link as MUILink } from '@material-ui/core';
+
+interface Props {
+  loggedIn: boolean;
+}
+
+const LogoHeader = ({ loggedIn }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.logo}>
+      <MUILink component={RouterLink} to={loggedIn ? '/dashboard' : '/login'}>
```

Comment:
Nice component! We don't need to do the conditional forwarding here, we can link to `/` or `/dashboard/ and have `ProtectedRoutes` (#28) take care of this for us.